packages/api-client/src/views/Request/ResponseSection/ResponseEmpty.vue

```diff
@@ -57,7 +94,7 @@ onBeforeUnmount(() => events.hotKeys.off(handleHotKey))
         <ScalarHotkey hotkey="↵" />
       </button>
       <button
-        v-if="!isReadOnly && layout === 'desktop'"
+        v-if="layout !== 'modal'"
```

Comment:
should we keep the desktop condition here? so we do not display the meta+n on web?

Reply:
feel free to adjust if you think it makes sense to do so

Reply:
ok will do! cause we can't use meta+n in web